common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -422,9 +422,16 @@ private ErrorStrings() {
     public final static String DEVICE_CODE_FLOW_EXPIRED_TOKEN_CODE = "expired_token";
     public final static String DEVICE_CODE_FLOW_EXPIRED_TOKEN_MESSAGE = "The token has expired, therefore authentication is no longer possible with this flow attempt. Re-run the Device Code Flow Protocol to try again.";
 
+    /**
+     * Device Code Flow only.
+     * The token was polled again after it was already received.
+     */
+    public final static String DEVICE_CODE_FLOW_INVALID_GRANT_CODE = "invalid_grant";
+    public final static String DEVICE_CODE_FLOW_INVALID_GRANT_MESSAGE = "The token for this device code has already been redeemed. To receive another access token, please re-run the Device Code Flow protocol.";
```

Comment:
Will this be shown to users? If yes, we'll need to have this localized

Reply:
I mean to say, this shouldn't even be something developers are _tempted_ to show users if it's not localized.

`devicecode_token_already_redeemed` for example, nobody would try and show that to a user. Whereas a full sentence worht of error may imply _maybe_ this is intended to be consumed by the user

Reply:
On further consideration, I think this is probably OK as is; developers who are concerned with localization should be aware _already_ that MSAL isn't localizing messages

Reply:
I'll self-resolve. Anyone feel free to reactivate if you thing that rationale is flawed